app/components/ExamPages/3-2_ScalesNotate.tsx

```diff
@@ -15,26 +15,38 @@ import { FormEvent, UserDataProps } from "@/app/lib/typesAndInterfaces";
 import { useState } from "react";
 import CardFooter from "../CardFooter";
 import NotateScale from "../NotateScale";
+import SnackbarToast from "../SnackbarToast";
 
 export default function ScalesNotation2({
   currentUserData,
   setCurrentUserData,
   nextViewState,
 }: UserDataProps) {
   const [scales, setScales] = useState<Array<string>>([]);
+  const [open, setOpen] = useState<boolean>(false);
+  const [isReady, setIsReady] = useState<boolean>(false);
```

Comment:
totally fine for now, but maybe rename isReady to isReadyForSubmit or something like that. I wasn't sure what it was doingl